python/llm/src/ipex_llm/transformers/npu_pipeline_model/convert_pipeline.py

```diff
@@ -182,8 +219,29 @@ def update_names_of_IR_and_export_blob(model, model_name, dir):
 
 def convert_llm(model: torch.nn.Module,
                 kv_len: int,
+                max_prompt_len: int,
                 transpose_value_cache: bool):
     if model.config.model_type == "llama":
+        # first token
+        from ipex_llm.transformers.npu_models.convert_mp import convert_forward
+        from ipex_llm.transformers.npu_models.llama_mp import gen_llama_fused_model_forward
+        from ipex_llm.transformers.npu_models.llama_mp import PrefillRunner
+        from transformers.models.llama.modeling_llama import LlamaModel
+
+        prefill_runner = PrefillRunner(
+            model,
+            max_output_len=kv_len,
+            max_prompt_len=max_prompt_len,
+            transpose_value_cache=transpose_value_cache,
+        )
+        llama_model_forward = gen_llama_fused_model_forward(
+            prefill_runner=prefill_runner, decode_runner=None
+        )
+        convert_forward(model, LlamaModel, llama_model_forward)
+        from transformers.models.llama.modeling_llama import LlamaForCausalLM
+        from ipex_llm.transformers.npu_models.llama_mp import llama2_casullm_forward
+        convert_forward(model, LlamaForCausalLM, llama2_casullm_forward)
```

Comment:
Refactor this in a common method to be called by both pipeline and accelerate models.

Reply:
Have refactored this part as you said. Would you mind taking a look again ?